make/autoconf/lib-freetype.m4

```diff
@@ -103,11 +107,15 @@ AC_DEFUN_ONCE([LIB_SETUP_FREETYPE],
   FREETYPE_TO_USE=bundled
   if test "x$OPENJDK_TARGET_OS" != "xwindows" && \
       test "x$OPENJDK_TARGET_OS" != "xmacosx" && \
-      test "x$OPENJDK_TARGET_OS" != "xaix"; then
+      test "x$OPENJDK_TARGET_OS" != "xaix" && \
+      test "x${STATIC_JAVA}" != "xtrue"; then
     FREETYPE_TO_USE=system
   fi
   if test "x$with_freetype" != "x" ; then
     if test "x$with_freetype" = "xsystem" ; then
+      if test "x${STATIC_JAVA}" = "xtrue"; then
+        AC_MSG_ERROR([--with-freetype=system does not work with --with-static-java=yes])
+      fi
```

Comment:
Why not ? Surely you aren't statically linking every library you find on the platform that's referenced by JDK  ?
I mean its fine to say that I'd prefer to statically link this library, but I don't understand the claim that this is the only thing that can work ?

Reply:
> Why not ? Surely you aren't statically linking every library you find on the platform that's referenced by JDK ?
> I mean its fine to say that I'd prefer to statically link this library, but I don't understand the claim that this is the only thing that can work ?

It's for providing libzip.a and libfreetype.a as part of the JDK 'static-libs' image. In our usages, I've learned that static linking is preferred whenever possible. We don't statically link libc.

Mine main reasoning/motivation for building and including the libzip.a and libfreetype.a in JDK 'static-libs' image is for ease of use. It reduces complications during the post hermetic image build process, if those libraries are need.